fontTesla.css

```diff
@@ -0,0 +1,28 @@
+/*! Generated by Font Squirrel (https://www.fontsquirrel.com) on April 27, 2020 */
+@font-face {
+  font-family: 'montserratregular';
+  src: url('fonts/montserrat-400-normal-webfont.woff2') format('woff2'), url('fonts/montserrat-400-normal-webfont.woff') format('woff');
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'montserratmedium';
+  src: url('fonts/montserrat-500-normal-webfont.woff2') format('woff2'), url('fonts/montserrat-500-normal-webfont.woff') format('woff');
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'montserratsemibold';
+  src: url('fonts/montserrat-600-normal-webfont.woff2') format('woff2'), url('fonts/montserrat-600-normal-webfont.woff') format('woff');
+  font-weight: 600;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'montserratbold';
+  src: url('fonts/montserrat-700-normal-webfont.woff2') format('woff2'), url('fonts/montserrat-700-normal-webfont.woff') format('woff');
+  font-weight: 700;
+  font-style: normal;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Tesla-12.04/pullRequest?prid=5512124)